public/app/plugins/panel/heatmap/utils.ts

```diff
@@ -257,6 +257,7 @@ export function prepConfig(opts: PrepConfigOpts) {
     return builder; // early abort (avoids error)
   }
 
+  // eslint-ignore @typescript-eslint/no-explicit-any
```

Comment:
@ashharrison90 @joshhunt any advice on how to avoid this one?

Reply:
Yup - you're running into the big type holes of grafana. I would say leave this in at the moment (It's a good marker for something that we would want to address later), and we can come up with a way to try and type field configs on data frames.

Reply:
My first pass at this would be to give DataFrame a generic to type the Field custom properties.